application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +136,32 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
+
+    private void attemptRetryReminder(Reminder reminder, Throwable failure) {
+        if (reminder.pendingReminders().getFailureAttempts() > MAX_FAILURE_RETRY) {
+            logger
+                .warn("""
+                        Failed to send a reminder with (authorID '{}') skipping it. This can be due to a network issue, \
+                        but also happen if the bot disconnected from the target guild and the \
+                        user has disabled DMs or has been deleted. It's (content includes '{}' with reminderId '{}'). The reason for (failure '{}')""",
+                        reminder.pendingReminders().getAuthorId(),
+                        reminder.pendingReminders().getContent(),
+                        reminder.pendingReminders().getId(), failure);
+            return;
+
+        }
+
+        int failureAttempts = reminder.pendingReminders().getFailureAttempts() + 1;
+        Instant remindAt = Instant.now().plusSeconds(60);
```

Comment:
isnt there a `plusMinutes(1)`?

Reply:
just `nanos`, `milis` and `seconds`

Reply:
so should i do this instead? i didn't know it was for mins

Reply:
oops, i meant to have 1 **minute** of delay